### DISCUSSION TITLE
Is it possible to make something like a search bar with debouncing?

### DISCUSSION THREAD

**Discussion by savicaleksa** [pacocoursey/cmdk#152] [Q&A]
I'm trying to make a search bar / combobox that shouldn't display any items on initial render, and display the filtered array after not pressing any keys for a brief delay.

I would at least want to not display the items after opening the popover for the first time because I have over 400 of them in my initial array and it takes like half a second to render them all.

Has anyone tried something similar?

**Comment by iPavlikov** [comment]
Hello, @savicaleksa! I have faced the same problem recently, and appeared that I have mistakenly passed controlled **value** prop to the **CommandList** component. If you are doing the same, just try to remove **value** prop from the **CommandList**.

**Comment by nythrox** [comment]
You can control the CommandList with `<Command shouldFilter={false}>`
